security/ldap.rst

```diff
@@ -519,12 +523,19 @@ Configuration example for form login and query_string
                         'service' => Ldap::class,
                         'dn_string' => 'dc=example,dc=com',
                         'query_string' => '(&(uid={username})(memberOf=cn=users,ou=Services,dc=example,dc=com))',
+                        'search_dn' => '...',
+                        'search_password' => 'the-raw-password',
                         // ...
                     ],
                 ],
             ]
         ]);
 
+.. deprecated:: 4.4
+
+    Using the ``query_string`` config without also defining a ``search_dn`` and
```

Comment:
... config option without defining search_dn and search_password is deprecated since ....